borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
fwiw i get almost every email from these review threads twice. Something broken with github it seems.